### ISSUE TITLE
Parse literals with double quotes

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss-labs/sparql-loader#54]
`57567.sparql` has a prefLabel with a properly escaped double quote in the literal. and we can load it directly in Blazegraph, but when we use the `sparql-loader` we see parsing errors.

**Comment by justinlittman** [comment]
```
INSERT DATA { GRAPH <http://sul.stanford.edu/rialto/graphs/wos> {
<http://sul.stanford.edu/rialto/agents/orgs/b87e0d339b9997a81c7078fc3c227133> <http://www.w3.org/2004/02/skos/core#prefLabel> "Stanford Global Studies Program's \"Urban Beyond Measure\" project" .
}};
INSERT DATA { GRAPH <http://sul.stanford.edu/rialto/graphs/wos> {
<http://sul.stanford.edu/rialto/agents/orgs/b87e0d339b9997a81c7078fc3c227133> <http://www.w3.org/2000/01/rdf-schema#label> "Stanford Global Studies Program's \"Urban Beyond Measure\" project" .
}};
```

**Comment by aaron-collier** [comment]
Fixed by #56
